weaviate/client.py

```diff
@@ -162,8 +162,8 @@ def __init__(
 
         Use this specific initializer when you want to create a custom Client specific to your Weaviate setup.
 
-        If you want to get going quickly connecting to WCS or a local instance then use the `weaviate.connect_to_wcs` or
-        `weaviate.connect_to_local` helper functions instead.
+        If you want to get going quickly connecting to the Weaviate Cloud or a local instance then use the
```

Comment:
Consider: 

To simplify connections to Weaviate Cloud or local instances, use the
        `weaviate.connect_to_weaviate_cloud` or `weaviate.connect_to_local` helper functions.